src/styles/index.css

```diff
@@ -0,0 +1,236 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.body {
+  color: #060b35;
+  margin: 0;
+}
+
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.main__card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  position: relative;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position-x: 19px;
+  background-position-y: 32px;
+  transition-duration: 300ms;
+}
+
+.main__card:hover {
+  transform: scale(1.2);
+}
+
+.card__title {
+  padding-inline: 16px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  position: absolute;
+  top: 206px;
+  transition-duration: 300ms;
+}
+
+.card__title:hover {
+  color: #34568b;
+}
+
+.card__product-code {
+  padding-inline: 16px;
+  position: absolute;
+  top: 246px;
```

Comment:
Absolute positioning for the product code can lead to layout problems. Evaluate if a more flexible layout approach could be beneficial.